.github/workflows/storybook.yml

```
@@ -0,0 +1,22 @@
+name: storybook-deploy
+on:
+    push:
+
+permissions: # so we can upload the storybook
+  contents: write
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)